esp/bindings/http/client/httpclient.cpp

```diff
@@ -951,9 +956,11 @@ HttpClientErrCode CHttpClient::postRequest(ISoapMessage &req, ISoapMessage& resp
     int statusCode = atoi(status.str());
     char statusClass = '0';
     if(status.length() > 0)
+    {
         statusClass = status.charAt(0);
+        errmsg.clear().appendf("HTTP Status %s", status.str());
```

Comment:
Very minor, but if you don't move the errmsg.clear() above the IF you could repeat an error log that was already logged inside the connect() above.